src/django/api/tests.py

```diff
@@ -1686,6 +1710,33 @@ def test_approved_facility_claim_info_is_in_details_response(self):
 
         self.assertEqual(response_data['description'], 'new_description')
 
+    @override_switch('claim_a_facility', active=True)
+    def test_updating_claim_profile_sends_email_to_contributor(self):
```

Comment:
Did we include a test for update notification but exclude a test for approval notification for a specific reason?

Reply:
Oh, the approval test is there-is but it's really subtle and probably could be denotated more clearly. It is essentially this line, which checks that there are now 2 emails in the outbox on approval rather than just 1:

https://github.com/open-apparel-registry/open-apparel-registry/pull/611/files#diff-a5f8ed705a64b1fe29a3eeb6439265d3R1406

Reply:
Rather than creating a new test I revised the test here to adjust its name and to add a comment about `len(mail.outbox) == 2`: 436301d0833a9aff795fb5ff81c35be02438e7a0